### DISCUSSION TITLE
PWA Starter

### DISCUSSION THREAD

**Discussion by lishaduck** [PSDTools/GPA_Calculator#2] [Ideas]
> Idea: why don't you just copy [pwa-builder/pwa-starter](https://github.com/pwa-builder/pwa-starter), and then copy this website in? Then it would just work.
> \- https://github.com/PSDTools/GPA_Calculator/commit/0455d58fe146e6813dfee222e580c6a7d6400a5e#commitcomment-106673309

If you don't mind, @ParkerH27, I think I'll do it.

Also, do you have third lunch? If so, do you mind if I come over[^1]?

[^1]: That sounds strange, but you know what I mean.

**Comment by PetalCat** [comment]
Yeah you can... I don't mind

**Comment by PetalCat** [comment]
im pretty sure i usually have 3rd lunch

**Reply by lishaduck** [reply]
I usually have 1st, except on "X" days.

**Comment by PetalCat** [comment]
I don't remember what I have on x days

**Reply by lishaduck** [reply]
Ok.
